### PR TITLE
Update Dependencies - Feb 2024

### PR DESCRIPTION
**Pull request by ThorntonMatthewD**
# Description
- Resolves the following security alerts:
   - [FastAPI Content-Type Header ReDoS](https://github.com/hackgvl/slack-events-bot/security/dependabot/9)
   - [aiohttp is vulnerable to directory traversal](https://github.com/hackgvl/slack-events-bot/security/dependabot/8)
   - [aiohttp's HTTP parser (the python one, not llhttp) still overly lenient about separators](https://github.com/hackgvl/slack-events-bot/security/dependabot/7)
- Updates a number of other dependencies
- Removes workaround for parsing request bodies in middleware being blocking now that the issue has been fixed in our dependencies.

### CODE REVIEW

File: src/server.py [hunk 1]
```diff
@@ -84,39 +84,12 @@ async def update_check_api_cooldown(team_domain: str | None) -> None:
...
-    return body
-
-
 @API.middleware("http")
 async def rate_limit_check_api(
     req: Request, call_next: Callable[[Request], Awaitable[None]]
 ):
     """Looks to see if /check_api has been run recently, and returns an error if so."""
-    req_body = await get_body(req)
+    req_body = await req.body()
```
Comment (oliviasculley) [inline src/server.py:92]: I wish there was a chef's kiss emoji for this!

**Review by oliviasculley — APPROVED** [review]
Time to reorganize hg labs into a logistics provider because we need to :ship: it!

File: pyproject.toml [hunk 2]
```diff
@@ -4,37 +4,27 @@ description = "An automated Slack bot to syndicate local events into Slack chann
 readme = "README.md"
 requires-python = ">=3.11"
 license = { text = "MIT" }
-version="0.1.0"
-authors = [
-  { name = "Olivia Sculley", email = "olivia@sculley.dev" },
-]
-keywords = [
-  "automation",
```
Comment (ThorntonMatthewD) [inline pyproject.toml:12]: I didn't mean to reformat this entire file. I was fighting with my autolinter hard, and I think it won.

File: src/server.py (hunk 1 above)
Comment (ThorntonMatthewD) [reply src/server.py:92]: It felt so good to rip that out, oh my gosh!

**Pull request merged**